src/interfaces/IFlashtestationRegistry.sol

```diff
@@ -11,22 +11,23 @@ import {WorkloadId} from "../utils/QuoteParser.sol";
 interface IFlashtestationRegistry {
     // TEE identity and status tracking
     struct RegisteredTEE {
-        WorkloadId workloadId; // The workloadID of the TEE device
+        bytes32 registrationHash; // Used as a downstream caching key
```

Comment:
Does this gain us much? Everywhere it's used we could use the `teeAddress` instead, assuming a 1:1 mapping between TEE workload and `teeAddress` (which we do assume is true). I think we can remove this and just use the `teeAddress` for the cache.